### DISCUSSION TITLE
[Question] .exe detected as a virus in windows

### DISCUSSION THREAD

**Discussion by alvaro2203** [flet-dev/flet#3129] [Q&A]
### Question

When i export my app to an .exe file with the flet pack command, windows detect my app like a virus and i can not share it.
How can i solve it?

**Comment by ndonkoHenri** [comment]
Flet pack command makes use of [pyinstaller](https://www.pyinstaller.org/).
I suggest you investigate from there...

**Comment by jjjunehkim** [comment]
In my case, after the anti-virus program stopped real-time scanning, the exe file was created and it worked normally.
Maybe you can try it
